packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts

```diff
@@ -0,0 +1,39 @@
+import { accordionDetailsClasses, accordionSummaryClasses } from "@mui/material";
+
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiAccordion: GetMuiComponentTheme<"MuiAccordion"> = (component, { palette }) => ({
+    ...component,
+    defaultProps: {
+        square: true,
+        disableGutters: true,
+        elevation: 0,
+        ...component?.defaultProps,
+    },
+    styleOverrides: mergeOverrideStyles<"MuiAccordion">(component?.styleOverrides, {
+        root: {
+            border: `solid ${palette.divider}`,
+            "&:not(:last-child)": {
+                marginBottom: "20px",
+            },
+            "&:before": {
+                display: "none",
+            },
+            [`& .${accordionSummaryClasses.expandIconWrapper}.${accordionSummaryClasses.expanded}`]: {
+                transform: "rotate(90deg)",
+            },
+            [`& .${accordionSummaryClasses.root}`]: {
+                display: "flex",
+                flexDirection: "row-reverse",
+                padding: "20px",
+                gap: "20px",
+                height: "86px",
```

Comment:
A fixed height-value will cause the text from overflowing when it requires multiple lines. 

<img width="1019" alt="image" src="https://github.com/vivid-planet/comet/assets/6264317/c16c31fe-10c4-4a41-be56-47d571a330ab">

Reply:
Good point, I removed the value